gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/MySqlDagManagementStateStoreTest.java

```diff
@@ -92,6 +95,16 @@ public static <T> boolean compareLists(List<T> list1, List<T> list2) {
     return true;
   }
 
+  @Test
+  public void testcanAcquireLeaseOnEntity() throws Exception{
```

Comment:
camel case typo... (but anyway, `canAcquireLeaseOnEntity` is not the name of the method)